cn_indian_payroll/cn_indian_payroll/overrides/payroll_configuration.py

```
@@ -1,119 +1,297 @@
-                    custom_field_doc.label = assignment.label
-
-                # if custom_field.hidden!=assignment.hidden:
-                #     custom_field.hidden=assignment.hidden
-
-                # if custom_field.allow_on_submit!=assignment.allow_on_submit:
-                #     custom_field.allow_on_submit=assignment.allow_on_submit
 
-                    
 
 
+def create_lta_component(name, salary_component, abbr, component_type, is_tax_applicable, is_reimbursement, data):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_lta_component`

Add return type annotation: `None`

(ANN201)